### ISSUE TITLE
*Suggestion* show only startdate when holiday duration is 1 day on Holiday widget  (code included)

### ISSUE THREAD

**Issue by Kahana82** [janizde/WP-Opening-Hours#40]
So I wanted to not have the holiday list to show the same date 2 times when the StartDate and the EndDate are equal. This happens when the holiday period is only 1 day.

I got this to work by modifying the following file:
/wp-content/plugins/wp-opening-hours/views/shortcode/holidays.php
```
...
<table class="op-table op-table-holidays">
  <tbody>
    <?php
    /** @var Holiday $holiday */
    foreach ($holidays as $holiday) :
    $highlighted = ($highlight && $holiday->isActive()) ? $class_highlighted : '';
    $strDateStart = Dates::format($date_format, $holiday->getDateStart());
    $strDateEnd = Dates::format($date_format, $holiday->getDateEnd());
    ?>
    <tr class="<?php echo $class_holiday; ?> <?php echo $highlighted; ?>">
      <td class="col-name"><?php echo $holiday->getName(); ?></td>     
	  <?php if($strDateStart == $strDateEnd){ ?>
			<td align="center" colspan="2" class="col-date-start"><?php echo $strDateStart; ?></td>
	  <?php } else { ?>
			<td class="col-date-start"><?php echo $strDateStart; ?></td>
			<td class="col-date-end"><?php echo $strDateEnd; ?></td>
	  <?php } ?>
    </tr>
    <?php endforeach; ?>
  </tbody>
</table>
...
```

**Comment by janizde** [comment]
thank you I will have a look at this for the next version

**Comment by foomep** [comment]
+1

**Comment by janizde** [comment]
@foomep implemented, will come in 2.1

**Comment by janizde** [comment]
@Kahana82 see above, didn't see you initially proposed that :)

**Comment by misagues** [comment]
Works in version 2.1 rc

But in the template "table" it is messy, it looks ugly.

![vista-tabla-opening-21](https://cloud.githubusercontent.com/assets/16360881/25648162/4764aac2-2fc9-11e7-9aa9-37cce234d5ed.JPG)

**Comment by janizde** [comment]
released in v2.1
